src/structures/Message.js

```diff
@@ -123,6 +125,12 @@ class Message extends Base {
      */
     this.embeds = (data.embeds || []).map(e => new Embed(e, true));
 
+    /**
+     * A list of component in the message e.g. ActionRows, Buttons
```

Comment:
This should be plural

```suggestion
     * A list of components in the message e.g. ActionRows, Buttons
```